nxdrive/gui/api.py

```diff
@@ -443,6 +445,29 @@ def default_server_url_value(self) -> str:
         """Make daily job better for our developers :)"""
         return getenv("NXDRIVE_TEST_NUXEO_URL", "")
 
+    @pyqtSlot(str, result=bool)
+    def free_disk_space_under_limit(self, path: str) -> bool:
+        free_space = self._free_disk_space(path)
+        return free_space < FREE_DISK_SPACE_LIMIT
+
+    @pyqtSlot(str, result=str)
+    def get_used_disk_space(self, uid: str) -> str:
+        engine = self._manager.engines.get(uid)
+        size = 0
+        if engine:
+            size = engine.dao.get_global_size()
+        return sizeof_fmt(size)
```

Comment:
The function is missing the translated suffix:
```suggestion
        return sizeof_fmt(size, suffix=Translator.get("BYTE_ABBREV"))
```